### PR TITLE
[13.0][FIX] delivery_tnt_oca: Set correct weight if not using packages

### PR DESCRIPTION
**Pull request by victoralmau**
Set correct weight if not using packages

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT40767

### CODE REVIEW

File: delivery_tnt_oca/models/tnt_request.py [hunk 1]
```diff
@@ -162,7 +162,11 @@ def _quant_package_data_from_picking(self):
             "LENGTH": data_total["length"],
             "HEIGHT": data_total["height"],
             "WIDTH": data_total["width"],
-            "WEIGHT": data_total["weight"],
+            "WEIGHT": (
+                data_total["weight"] / self.record.number_of_packages
```
Comment (pedrobaeza) [inline delivery_tnt_oca/models/tnt_request.py]: Shouldn't it be always the division?

**Comment by pedrobaeza** [comment]
But this has to be confirmed by the carrier that it should be that way, isn't it?

File: delivery_tnt_oca/models/tnt_request.py (hunk 1 above)
Comment (victoralmau) [reply delivery_tnt_oca/models/tnt_request.py]: Yes, you are right, because it is not sent in PACKAGE the package listing (if used).

**Comment by pedrobaeza** [comment]
It's a bit weird that we send the weight per package in the totals.

**Comment by pedrobaeza** [comment]
OK, confirmed:

```
En la request de EC el peso que vuelca a nuestros sistemas es el del apartado PACKAGE y este se debe especificar el peso total como peso total dividido por el número de ITEMS por ejemplo:

        <PACKAGE>
          <ITEMS>2</ITEMS>
          <DESCRIPTION>MAY\OUT\78475</DESCRIPTION>
          <LENGTH>0.35</LENGTH>
          <HEIGHT>0.35</HEIGHT>
          <WIDTH>0.35</WIDTH>
          <WEIGHT>10</WEIGHT>
        </PACKAGE>

En este caso el peso total del envío es 20 KG.
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Let's fast track it to fix the problem:

/ocabot merge patch

**Pull request merged**